src/pages/trade-types/cfds/_start-trading.tsx

```diff
@@ -15,34 +16,40 @@ const StartTrading = () => {
                 <SmallContainer direction="column" ai="flex-start">
                     <Show.Desktop>
                         <Header as="h3" type="section-title" mb="4rem">
-                            {localize('Start trading CFDs on Deriv')}
+                            {show_branding && localize('Start trading CFDs on Deriv')}
                         </Header>
                     </Show.Desktop>
                     <Show.Mobile>
                         <Header as="h3" type="section-title" mb="4rem">
-                            <Localize
-                                translate_text="Start trading CFDs on <0/>Deriv<0/>"
-                                components={[<br key={0} />]}
-                            />
+                            {show_branding && (
+                                <Localize
+                                    translate_text="Start trading CFDs on <0/>Deriv<0/>"
+                                    components={[<br key={0} />]}
+                                />
+                            )}
                         </Header>
                     </Show.Mobile>
 
                     <Timeline>
                         <Timeline.Item title={<Localize translate_text="Practise" />}>
                             {is_eu_country
-                                ? localize(
+                                ? show_branding &&
+                                  localize(
                                       'Open a demo CFDs account and practise with an unlimited amount of virtual funds.',
                                   )
-                                : localize(
+                                : show_branding &&
+                                  localize(
                                       'Open a demo Deriv MT5 (DMT5) or demo Deriv X account and practise with an unlimited amount of virtual funds.',
                                   )}
                         </Timeline.Item>
                         <Timeline.Item title={<Localize translate_text="Trade" />}>
                             {is_eu_country
-                                ? localize(
+                                ? show_branding &&
+                                  localize(
                                       'Trade with a real CFDs account. Get access to leverage and trade positions larger than your existing capital.',
                                   )
-                                : localize(
+                                : show_branding &&
+                                  localize(
                                       'Trade with a real Deriv MT5 (DMT5) or real Deriv X account and get access to high leverage to trade positions larger than your existing capital.',
                                   )}
                         </Timeline.Item>
```

Comment:
how about we render these two items conditionaly:
```suggestion
{show_branding && <Timeline.Item> .... </Timeline.Item>}
```